api/numaresourcesoperator/v1/numaresourcesoperator_types.go

```diff
@@ -210,9 +210,10 @@ func init() {
 }
 
 func (ngc *NodeGroupConfig) ToString() string {
```

Comment:
why remove the `ngc != nil` check?

Reply:
by mistake, we do need it back. Thanks for catching